test/e2e/bluegreen_test.go

```diff
@@ -158,3 +158,121 @@ spec:
 		Then().
 		ExpectRevisionPods("revision 2 has active metadata2", "2", podsHaveActiveMetadata2)
 }
+
+func (s *BlueGreenSuite) TestBlueGreenProgressDeadlineExceededWithPause() {
+	s.Given().
+		RolloutObjects(`
+apiVersion: argoproj.io/v1alpha1
+kind: Rollout
+metadata:
+  name: rollout-bluegreen
```

Comment:
Can we use a unique name for the rollout which is descriptive for the test? It helps with grepping controller logs during debugging